web-backend/src/main/java/com/statusneo/vms/controller/VisitorController.java

```diff
@@ -108,9 +109,9 @@ public ResponseEntity<String> validateOtp(
             return ResponseEntity.badRequest().body("Invalid visit ID");
         }
 
-        boolean isValid = otpService.validateOtp(visit, otp);
+        VerificationResult result = otpService.validateOtp(visit, otp);
```

Comment:
Are we calling validateOTP form here or from visitService? Only one of them should be kept.